src/Nethermind/Nethermind.Init/Steps/InitializeNetwork.cs

```diff
@@ -559,6 +559,13 @@ private async Task InitPeer()
         NodeRecordSigner nodeRecordSigner = new(_api.EthereumEcdsa, new PrivateKeyGenerator().Generate());
         EnrRecordParser enrRecordParser = new(nodeRecordSigner);
         EnrDiscovery enrDiscovery = new(enrRecordParser, _api.LogManager); // initialize with a proper network
+
+        if (_networkConfig.DisableDiscV4DnsFeeder)
```

Comment:
Looks like it runs if disabled.
Maybe rename to `Enable...` with default value `true`?

Reply:
Ooff.. my bad.